demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -148,6 +150,35 @@ def is_insecure_configured_correctly(self):
             return self.is_valid_param(insecure_field_name, 'Trust any certificate (not secure)')
         return True
 
+    def is_checkbox_param_configured_correctly(self):
+        # type: () -> bool
+        """Check that if an integration has a checkbox parameter it is configured properly.
+        Returns:
+            bool. True if the checkbox parameter is configured correctly, False otherwise.
+        """
+        configuration = self.current_file.get('configuration', [])
+        for configuration_param in configuration:
+            param_name = configuration_param['name']
+            if configuration_param['type'] == 8 and param_name not in ('insecure', 'unsecure', 'proxy', 'isFetch'):
+                self.is_valid_checkbox_param(configuration_param, param_name)
+        if not self.is_valid:
+            return False
+
+        return True
+
+    def is_valid_checkbox_param(self, configuration_param, param_name):
+        # type: (dict, str) -> None
+        """Check if the given checkbox parameter required field is False.
+        Returns:
+            str. Empty string if valid, error message otherwise.
+        """
+        err_msg = None
+        if configuration_param.get('required', False):
+            err_msg = Errors.wrong_required_value(param_name)
+        if err_msg:
+            self.is_valid = False
```

Comment:
better to return true false then to update this param

Reply:
done